pkg/web_app/lib/src/account.dart

```diff
@@ -52,10 +52,12 @@ void _initSessionMonitor() {
     final authenticatedAt = status.authenticatedAt;
     String? displayMessage;
     if (expires == null || authenticatedAt == null) {
-      displayMessage = 'Your session has expired.';
+      displayMessage =
+          'Your session has expired: you may need to reload the page to resume editing.';
```

Comment:
Could it be?
```suggestion
          'Your admin session has expired: you may need to reload the page to resume editing.';
```

Reply:
Don't you **have** to reload?
```suggestion
          'Your session has expired: reload the page before making further administrative actions.';
```